network-area-diagram/src/main/java/com/powsybl/nad/layout/Layout.java

```diff
@@ -25,4 +26,8 @@ public interface Layout {
     Map<String, Point> getInitialNodePositions();
 
     Set<String> getNodesWithFixedPosition();
+
+    void setTextNodesWithFixedPosition(Map<String, TextPosition> textNodesWithFixedPosition);
+
+    Map<String, TextPosition> getTextNodesWithFixedPosition();
```

Comment:
I think you can remove the getter (the others getters should not be in the interface as well, but that's not the topic of this PR)

Reply:
getter removed from the interface